javamelody-core/src/main/java/net/bull/javamelody/internal/model/Collector.java

```diff
@@ -313,7 +325,7 @@ public void collectLocalContextWithoutErrors() {
 		// car on n'en a pas besoin pour la collecte et cela économise des requêtes sql
 		try {
 			final JavaInformations javaInformations = new JavaInformations(
-					Parameters.getServletContext(), false);
+					Parameters.getServletContext(), includeDetails);

```

Comment:
`new JavaInformations(Parameters.getServletContext(), true)` does too much each minute than what is needed here. Too much overhead is currently added there for this feature.

Reply:
I think that the stack-traces of concerned threads only should be fetched using `Thread.getStackTrace()`, using a thread in a transient field in CounterRequestContext.